server/boardData.js

```diff
@@ -174,7 +174,7 @@ class BoardData {
         this.addChild(message.parent, message);
         break;
       case "clear":
-        if(jwtauth.checkIfModerator(message.token)) {
+        if(jwtBoardnameAuth.roleInBoard(message.token) === "moderator") {
```

Comment:
Shouldn't we check that the user is a moderator *in the current board* ? We now have roles like `moderator:board77`

Reply:
It depends on the concept. If we are going to be backwards compatible, I don't think it is needed as all moderators have access to the Clear tool, which will remove all content from the board. For example, if moderators do not have a board assigned to them, they can remove all content from each board. On the other hand, if the moderators have a board assigned to them, they can only remove a content from the board assigned to them, because they do not have the right to access the boards of other moderators who have the board assigned to them.

Reply:
The Function returns the role moderator, when The board name is correct with the board name in the url, and returns moderator when there is set the role moderator in the roles claim but no boardname is set. in this case a user with such a JWT can wipe every board.

Reply:
Yes, you have to give the board name to the roleInBoard function, and it will do the right thing if the role is set to just `"moderator"`.